src/handlers/InteractionCommandHandler.ts

```diff
@@ -4,28 +4,28 @@ import { CommandContext } from '../lib/structures/contexts/CommandContext';
 import { Handlers } from '../lib/managers/HandlerManager';
 import { Commands } from '../lib/managers/CommandManager';
 import { setTimeout } from 'node:timers';
-import { Logger, Events } from '../lib/util/logger/Logger';
+import { Events, Logger } from '../lib/util/logger/Logger';
 import { Util } from '../lib/util/Util';
+import { isErr } from '@sapphire/result';
 
 const cooldowns = new Collection<string, Collection<string, number>>();
 
 export async function InteractionCommandHandler(interaction: CommandInteraction | ContextMenuInteraction) {
 	const client = interaction.client as GClient;
 
 	const command = Commands.get(interaction.commandName);
-	if (!command)
-		return client.options?.unknownCommandMessage ? interaction.reply({
-			content: (await Util.getResponse('NOT_FOUND', { client })),
-		}) : null;
+	if (!command && client.options.unknownCommandMessage)
```

Comment:
Return it as it was before, otherwise it throws an error on a bad command when you don't have `unknownCommandMessage` enabled.